src/command/dev/mcp.rs

```diff
@@ -1,90 +1,18 @@
 use camino::Utf8PathBuf;
 use clap::Parser;
 use serde::Serialize;
-use strum_macros::Display;
 
 pub mod binary;
 pub mod install;
 pub mod run;
 
-#[derive(clap::ValueEnum, Clone, Default, Debug, Display, Serialize)]
-enum MutationMode {
-    /// Don't allow any mutations
-    #[default]
-    None,
-    /// Allow explicit mutations, but don't allow the LLM to build them
-    Explicit,
-    /// Allow the LLM to build mutations
-    All,
-}
-
 #[derive(Debug, Clone, Serialize, Parser)]
 pub struct Opts {
     /// Enable the MCP server
     #[arg(long = "mcp")]
     pub enabled: bool,
```

Comment:
Should users specify both `--mcp` and `--mcp-config`?

Reply:
The issue is that the config file is optional since it includes defaults for all options. If we make it so that you always need to have a config to run the MCP server, that might make the quickstart more complicated than it needs to be. I can update this PR to look at if either one is specified before starting the MCP server, if that works